brambling/models.py

```diff
@@ -1098,6 +1098,10 @@ def get_eventhousing(self):
     def has_dwolla_payments(self):
         return self.transactions.filter(method=Transaction.DWOLLA).exists()
 
+    def awaiting_check_confirmation(self):
```

Comment:
Okay, cool. In that case I'm bouncing this PR back to @chigby for more work. I could also give it a swing, but I think @chigby is still less busy than me. :stuck_out_tongue_closed_eyes: